plugins/ImageReader/ConfigUI.qml

```diff
@@ -143,6 +143,51 @@ UM.Dialog
             }
         }
 
+        UM.TooltipArea {
+            Layout.fillWidth:true
+            height: childrenRect.height
+            text: catalog.i18nc("@info:tooltip","For lithophanes a simple logarithmic model for translucency is available. For height maps the pixel values correspond to heights linearly.")
+            Row {
+                width: parent.width
+
+                Label {
+                    text: "Color Model"
+                    width: 150 * screenScaleFactor
+                    anchors.verticalCenter: parent.verticalCenter
+                }
+                ComboBox {
+                    id: color_model
+                    objectName: "ColorModel"
+                    model: [ catalog.i18nc("@item:inlistbox","Translucency"), catalog.i18nc("@item:inlistbox","Linear") ]
+                    width: 180 * screenScaleFactor
+                    onCurrentIndexChanged: { manager.onColorModelChanged(currentIndex) }
+                }
+            }
+        }
+
+        UM.TooltipArea {
```

Comment:
The transmittance textfield is not used for the linear model right? Then `visible` for this item should depend on the `color_model`

Reply:
I can't get it to work. I'm not even sure it's possible to change the layout of the dialog box depending on some variable automatically. 

Or do you know how it's done?

I don't think it's really important, though.

Reply:
Thanks for resolving it.